app/utils/WoWFilers/expansions.ts

```
@@ -0,0 +1,32 @@
+  { label: 'The War Within', value: '11' }
+]
+
+/**
+ * Gets the expansion name from its number
+ * @param expansionNumber The expansion number to look up
+ * @returns The expansion name or "All" for -1, or "Unknown Expansion" if not found
+ */
+export const getExpansionName = (expansionNumber: number): string => {
+  if (expansionNumber === -1) return 'All'
+  const expansion = expansionOptions.find(
+    (exp) => parseInt(exp.value) === expansionNumber
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)